index.html

```diff
@@ -903,10 +903,29 @@ <h4>Issue Credential</h4>
         </p>
         <p>
 If a provided `credential` already contains one or more proofs,
-an issuing instance SHOULD append its proofs to the list of existing proofs
-provided by the caller, converting a single proof value to an array of
-proof values if necessary. An issuing instance MAY be configured to return an
-error if `credential` values that contain existing `proof` values are provided.
+the behavior is determined by the issuer instance configuration. An issuing instance
+SHOULD be configured to handle existing proofs in one of the following ways:
+        </p>
+        <ul>
+          <li>
+            <strong>Proof Sets</strong>: Append their proofs to the list of existing proofs 
+provided by the caller, converting a single proof value to an array of proof values 
+if necessary, creating a proof set where multiple proofs exist in parallel.
```

Comment:
I think this needed slight adjustment to make it clearer that a proof set is used whether the existing list is augmented or a new list of proofs is created and to adjust the "their" language which seemed a little off to me.

```suggestion
            <strong>Proof Sets</strong>: Append new proofs to the list of existing proofs 
provided by the caller, first converting any existing single proof to a list, if necessary.
Here there is no binding to any existing proofs; the new proofs exist in parallel with
those previously provided by the caller.
```